### PR TITLE
refactor: replace config command with login and context

### PR DESCRIPTION
**Pull request by aiwantaozi**
**Problem:**
1. cli command config is not clear enough
2. environment start/stop lack of arg

**Solution:**
1. use login and context to replace config
2. environment start/stop lack of arg

**Related Issue:**
#1790 
#1761

### CODE REVIEW

File: pkg/cli/api/version.go [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+
+// Version include the version and commit.
+type Version struct {
+	Version      string `json:"version" yaml:"version"`
+	GitCommit    string `json:"gitCommit" yaml:"gitCommit"`
+	IsDevVersion bool   `json:"isDevVersion" yaml:"isDevVersion"`
+}
+
+// GetVersion get client, server and openapi version.
+func GetVersion(sc *config.Config) (*Info, error) {
```
Comment (alexcodelf) [inline pkg/cli/api/version.go:28]: The func name is confused, get version but return info

Comment (aiwantaozi) [reply pkg/cli/api/version.go]: rename to VersionInfo

**Review by thxCode — APPROVED** [review]
LGTM

**Review by alexcodelf — APPROVED** [review]
LGTM

**Pull request merged**